### DISCUSSION TITLE
Unable to clone the repo

### DISCUSSION THREAD

**Discussion by optimax** [ElinamLLC/SharpVectors#286] [General]
It looks like the repo is corrupted. I don't get that with other GitHub projects.

![image](https://github.com/ElinamLLC/SharpVectors/assets/832703/205fdb83-8e78-45ee-8b69-07b06761c740)

I tried from several locations, so it's not my connection.

**Comment by paulushub** [comment]
Sorry, I have tried it on two other PCs and it worked without any issue.
Your issue may be related to the following:
https://stackoverflow.com/questions/66206530/rpc-failed-curl-56-failure-when-receiving-data-from-the-peer

Please search for the error: **RPC failed: curl 56 failure when receiving data from the peer**
